library/js/slidein-close.js

```diff
@@ -44,6 +44,7 @@
 			setTimeout( function() {
 				slidein.removeClass( 'hustle-show' );
 				content.removeClass( 'hustle-animate-out' );
+				HUI.trapFocus();
```

Comment:
@creador-dev, please, explain this part. Why call the trap function after the modal closes? Shouldn't this be `untrapFocus` function instead?

Reply:
@iamleigh If we have more modals then we have to call this to change trap focus onto the next modal.

Reply:
@creador-dev, then I guess you're adding some checks in case there's only one modal, to untrap focus instead. Correct?